src/PolyhedralGeometry/triangulations.jl

```diff
@@ -442,7 +442,7 @@ function secondary_polytope(P::Polyhedron{T}) where T<:scalar_types
 end
 
 @doc Markdown.doc"""
-    isregular(pts::Union{SubObjectIterator{<:PointVector}, AbstractMatrix, Oscar.MatElem},cells::Vector{Vector{Vector{Int64}}})
+    isregular(pts::MatrixUnion, cells::Vector{Vector{Vector{Int64}}})
 
 Compute whether a triangulation is regular.
```

Comment:
Off-topic, but: I really would expect the argument to be a triangulation then (i.e. and object of some type `Triangulation`